### PR TITLE
Equinix bot (WIP)

### PR DESCRIPTION
**Pull request by APiligrim**

### CODE REVIEW

**Comment by jimmylee** [comment]
Very cool :-)

**Comment by jimmylee** [comment]
A question for you @APiligrim, should this live in this repo or should it live in a separate repo? Seems like the bot is its own service,

Might be good to just use https://github.com/application-research/next-sass and spin out a separate service?

File: common/constants.ts [hunk 1]
```diff
@@ -115,13 +114,29 @@ export const staticEnvironmentPayload = {
       Uuid: '8ceea3cd-7608-4428-8d6b-99f2acc80ce3',
       Name: 'Shuttle-8',
     },
+    {
+      Uuid: 'd2962032-37e4-43f5-ad9b-15535c5555d6',
+      Name: 'Shuttle-9',
```
Comment (APiligrim) [inline common/constants.ts:119]: asked Luc for updated Shuttle uuids & will update them for the latest ones before merging

**Comment by APiligrim** [comment]
will add it to a separate service ^

**Pull request closed without merging**